### PR TITLE
Implement plt()

### PR DESCRIPTION
**Pull request by makotoshimazu**
Implement plt() method to get an address to the plt's instruction pointer. The pwntools in python is using unicorn so it's used for this PR too.
However, currently the upstream requires libunicorn in your system and "unicorn" in crates.io is deprecated since it's merged to the mainstream. 
@Hakuyume made a pull request and tentatively this PR is depending on that branch.

### CODE REVIEW

File: examples/plt.rs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "jmp got[1]",
+  "load ebx, got[2]",  // addr_plt2
+  "je ebx 0 0xNNNN",
+  "jmp got[2]",
+  "load ebx, got[3]",
+  "je ebx 0 0xNNNN",
+  "jmp got[3]",
+]
+
+// これがほしい
```
Comment (Hakuyume) [inline examples/plt.rs:47]: Do you need this comment?

**Comment by S-Nakamur-a** [comment]
I know it seems like an unimportant detail, but the name of rust package registry is crates.io , not crate.io.

File: Cargo.toml [hunk 2]
```diff
@@ -18,6 +18,10 @@ version = "0.2.0"
 elf-utilities = "=0.2.8"
 num-derive = "0.3"
 num-traits = "0.2"
+unicorn = { git = "https://github.com/Hakuyume/unicorn", branch = "request-lgtm" }
+# TODO: Once https://github.com/unicorn-engine/unicorn/pull/1447 is merged to trunk and published, use the official unicorn crate.
+# unicorn = { path = "../unicorn/bindings/rust" }
```
Comment (S-Nakamur-a) [inline Cargo.toml:23]: i think this line should be removed.

**Comment by yukidmy** [comment]
LGTM!!!!!!!!!! Great contribution!!!!!!!!!!

File: examples/plt.rs (hunk 1 above)
Comment (makotoshimazu) [reply examples/plt.rs]: Thanks, updated the comment.

File: Cargo.toml (hunk 2 above)
Comment (makotoshimazu) [reply Cargo.toml]: thx, removed.

**Review by yukidmy — APPROVED** [review]
Approved!!!!!!!!!!

File: examples/plt.rs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    let opts: Opts = Opts::parse();
+    let pwn = Pwn::new(&opts.elf_file);
+    println!("name: printf, addr: 0x{:x}", pwn.plt("printf").unwrap());
+
+    Ok(())
+}
+
+/*
+
+got: []u64 = [
```
Comment (S-Nakamur-a) [inline examples/plt.rs:20]: feel like golang

**Review by Hakuyume — APPROVED** [review]
!!!

**Comment by S-Nakamur-a** [comment]
i did'nt know "tentatively" ... this PR was a very good lesson for me.

**Pull request merged**